modules/news/plugins/home.php

```diff
@@ -26,8 +26,6 @@
 if ($db->num_rows($query) > 0) {
     while ($row = $db->fetch_array($query)) {
         $page = floor(($row['comments'] - 1) / 20);
-        $smarty->assign('link', "index.php?mod=board&action=thread&tid={$row['tid']}&posts_page={$page}#pid{$row['pid']}");
```

Comment:
Because there are two link assignments and I removed the wrong (board) one. The `link` got overwritten anyway. Means the removed line was "dead code".
See
https://github.com/lansuite/lansuite/blob/815d8a09b05d78fee76340d878868b51d3de86ad/modules/news/plugins/home.php#L29-L31